src/pagingscrollview/src/NIPagingScrollView.h

```diff
@@ -68,7 +68,11 @@ typedef enum {
 
 #pragma mark Configuring Presentation
 
+// Controls the border between images.
```

Comment:
s/images/pages/

Also may want to mention that the page margin is only applied in the paging direction.